tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -25,6 +26,10 @@ class Contacts(Base):
     _group_container_selector = "#groups-container"
     _contact_locator = (By.CSS_SELECTOR, 'li[data-uuid]:not([data-group="ice"])')
 
+    @property
+    def manifest_url(self):
```

Comment:
Nit: We don't need to define the manifest_url as a property here. This can be a class variable. The main reason is: `self.DEFAULT_PROTOCOL` and `self.DEFAULT_APP_HOSTNAME` are already defined.